emails/tests/views_tests.py

```diff
@@ -1125,6 +1140,77 @@ def test_complaint_log_with_optout(self) -> None:
         assert log_data["user_match"] == "found"
         assert not log_data["fxa_id"]
 
+    @override_flag("disable_mask_on_complaint", active=True)
+    def test_complaint_disables_mask(self):
+        """
+        A notificationType of complaint:
+            1. sets enabled=False on the mask, and
+            2. returns 200.
+        """
+        self.ra = baker.make(
+            RelayAddress, user=self.user, address="ebsbdsan7", domain=2
+        )
+
+        # The top-level JSON object for complaints includes a "mail" field
+        # which contains information about the original mail to which the notification
+        # pertains. So, add a "mail" field with content from our russian_spam fixture
+        russian_spam_notification = create_notification_from_email(
+            EMAIL_INCOMING["russian_spam"]
+        )
+        spam_mail_content = json.loads(
+            russian_spam_notification.get("Message", "")
+        ).get("mail", {})
+        complaint_body_message = json.loads(self.complaint_body["Message"])
+        complaint_body_message["mail"] = spam_mail_content
+        complaint_body_with_spam_mail = {"Message": json.dumps(complaint_body_message)}
+        assert self.ra.enabled is True
+
+        response = _sns_notification(complaint_body_with_spam_mail)
+        assert response.status_code == 200
+
+        self.ra.refresh_from_db()
+        source = self.mock_ses_client.send_raw_email.call_args.kwargs["Source"]
+        destinations = self.mock_ses_client.send_raw_email.call_args.kwargs[
+            "Destinations"
+        ]
+        raw_message = self.mock_ses_client.send_raw_email.call_args.kwargs["RawMessage"]
+        data_without_newlines = raw_message["Data"].replace("\n", "")
+
+        assert self.ra.enabled is False
```

Comment:
```suggestion
        assert self.user.profile.auto_block_spam is True
        assert self.ra.enabled is False
```
(suggestion, non-blocking) To make sure that the auto_block_spam still works as expected.